src/objective-c/tests/CoreTests/Podfile

```diff
@@ -5,6 +5,7 @@ install! 'cocoapods', :deterministic_uuids => false
 GRPC_LOCAL_SRC = '../../../..'
 
 def grpc_deps
+  pod 'BoringSSL-GRPC', :podspec => "#{GRPC_LOCAL_SRC}/src/objective-c"
```

Comment:
is CoreTest buildable w/o adding the boringssl pod ?  ideally CoreTest only pull in gRPC-Core pod

Reply:
I tried, unfortunately not buildable. My guess is because boringssl pod hasn't published in cocoapods repo yet.